core/src/main/java/org/infinispan/security/audit/LoggingAuditLogger.java

```diff
@@ -21,10 +24,26 @@
 public class LoggingAuditLogger implements AuditLogger {
    static final AuditMessages auditLog = Logger.getMessageLogger(AuditMessages.class, "org.infinispan.AUDIT");
 
+   volatile InfinispanTelemetry telemetryService;
+
+   public void setTelemetryService(InfinispanTelemetry telemetryService) {
+      this.telemetryService = telemetryService;
+   }
+
    @Override
    public void audit(Subject subject, AuditContext context, String contextName, AuthorizationPermission permission,
          AuditResponse response) {
       auditLog.auditMessage(response, Security.getSubjectUserPrincipal(subject), permission, context, contextName);
+
+      if (telemetryService != null) {
+         String cacheName = (AuditContext.CACHE.equals(context)) ? contextName : context.toString();
+
+         InfinispanSpanAttributes attributes = new InfinispanSpanAttributes.Builder(SpanCategory.SECURITY)
+               .withCacheName(cacheName)
+               .build();
+
+         telemetryService.startTraceRequest(response.name(), attributes).complete();
```

Comment:
what are we tracing here? the span is started and completed right away.

Reply:
Nothing wrong in my opinion. In this case we don't have a real start time / end time. We want only to trace the event indicating the time of the event and not its duration.

Reply:
but why? logs already have a timestamp so why do we need an OpenTelemetry event saying "hey, I write to a log file"? Is this a product security requirement @tristantarrant ?